DSL/Aggregation/HistogramAggregation.php

```diff
@@ -0,0 +1,177 @@
+<?php
+
+/*
+ * This file is part of the ONGR package.
+ *
+ * (c) NFQ Technologies UAB <info@nfq.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace ONGR\ElasticsearchBundle\DSL\Aggregation;
+
+use ONGR\ElasticsearchBundle\DSL\Aggregation\Type\BucketingTrait;
+
+/**
+ * Class representing Histogram aggregation.
+ */
+class HistogramAggregation extends AbstractAggregation
+{
+    use BucketingTrait;
+
+    const DIRECTION_ASC = 'asc';
+    const DIRECTION_DESC = 'desc';
+
+    /**
+     * @var int
+     */
+    private $interval;
+
+    /**
+     * @var int
+     */
+    private $minDocCount;
+
+    /**
+     * @var array
+     */
+    private $extendedBounds;
+
+    /**
+     * @var string
+     */
+    protected $orderMode;
+
+    /**
+     * @var string
+     */
+    protected $orderDirection;
+
+    /**
+     * @var bool
+     */
+    private $keyed;
+
+    /**
+     * @return bool
+     */
+    public function isKeyed()
+    {
+        return $this->keyed;
+    }
+
+    /**
+     * @param bool $keyed
+     */
+    public function setKeyed($keyed)
+    {
+        $this->keyed = $keyed;
+    }
+
+    /**
+     * @param string $mode
+     * @param string $direction
+     */
+    public function setOrder($mode, $direction = self::DIRECTION_ASC)
```

Comment:
Please provide a description in docBlock for this method also.

*P.S. this is not a test method :)*